base/clouddriver/base/kustomization.yml

```diff
@@ -3,3 +3,6 @@ kind: Kustomization
 resources:
 - deployment.yml
 - service.yml
+commonLabels:
```

Comment:
> The advantage is less duplication of labels (and reducing copy/paste
mistakes of forgetting to update one label). The disadvantage is that
it's a bit less clear exactly what labels get generated (ie, we no longer
explicitly include a selector, etc.)

🎉 I think this is so much better! Definitely much more likely that there will be a copy/paste mistake than any issues understanding what selector gets generated (which most users shouldn't really care about anyway?).